packages/gatsby-source-drupal/src/normalize.js

```diff
@@ -6,6 +6,7 @@ const nodeFromData = (datum, createNodeId) => {
     id: createNodeId(datum.id),
     drupal_id: datum.id,
     parent: null,
+    drupal_parent: attributes.parent,
```

Comment:
Based on the discussion, 

- `drupal_parent` can't be a Node because `attributes.parent` doesn't always have an `id`
- `drupal_children` is out of scope of this PR

I would recommend that we name this `drupal_parent_id` for now and call it a day for this particular issue. 

What do you think @xaviemirmon @pieh?